modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-service/src/main/resources/com/liferay/dynamic/data/mapping/dependencies/ddm/text-html.ftl

```diff
@@ -7,12 +7,22 @@
 	inputEditorName = "${namespacedFieldName}Editor"
 
 	fieldValue = paramUtil.getString(request, "${inputEditorName}", fieldValue)
+
+	toolbarSet = ""
```

Comment:
Hey @dustinryerson, will it break if we just set the default value to "liferay" and remove the last `<#else>`  below?